### PR TITLE
Update tests to use new test helpers

### PR DESCRIPTION
**Pull request by cadeParade**
## Description
Update our tests to use the new Ember test helpers following: 
https://github.com/emberjs/rfcs/blob/master/text/0232-simplify-qunit-testing-api.md
https://github.com/emberjs/rfcs/blob/master/text/0268-acceptance-testing-refactor.md

The main structural changes were made in `setup-acceptance.js` and `test-helper.js`. The rest of the changes were updating syntax to match the new test helpers 
- use native DOM selectors rather than jquery specific selectors (i.e. can no longer use `:contains(foo)` selector)
- Replace `currentPath` with `currentRouteName`
- Import `click`, `fillIn`, etc helpers from `ember-test-helpers`
- `await` everything
- Upgrade to ember-page-object v1.15.0-beta.3

### CODE REVIEW

File: tests/acceptance/organization-test.js [hunk 1]
```diff
@@ -21,34 +25,34 @@ describe('Acceptance: Organization', function() {
...
       await percySnapshot(this.test);
     });
 
     it('can create new organization and update org switcher', async function() {
-      await visit(`/${this.organization.slug}`);
-      await click('.OrganizationsSwitcherNav-item');
-      await click('a:contains("Create new organization")');
-      expect(currentPath()).to.equal('organizations.new');
+      await ProjectPage.visitOrg({orgSlug: this.organization.slug});
+      await click('[data-test-toggle-org-switcher]');
```
Comment (maprules1000) [inline tests/acceptance/organization-test.js]: 🙌

File: tests/integration/components/projects/browser-family-selector-test.js [hunk 2]
```diff
@@ -92,6 +93,9 @@ describe('Integration: BrowserFamilySelector', function() {
     });
   });
 
+  // These tests don't wait on the add/remove browser action to finish. Not sure why, but
+  // `await settled()` seems to help...
+  // Perhaps this issue is helpful
```
Comment (maprules1000) [inline tests/integration/components/projects/browser-family-selector-test.js]: did you mean to link to a gh issue here?

File: tests/integration/components/repo-refresh-status-test.js [hunk 3]
```diff
@@ -1,37 +1,33 @@
...
 import {describe, it} from 'mocha';
-import {setupComponentTest} from 'ember-mocha';
+import {setupRenderingTest} from 'ember-mocha';
 import hbs from 'htmlbars-inline-precompile';
 import moment from 'moment';
-
-// FIXME: (faun) update these to use native dom selectors once
-// the latest testing APIs are available in this app.
-// https://guides.emberjs.com/release/testing/testing-component
-// https://github.com/emberjs/ember-mocha#setup-tests
```
Comment (maprules1000) [inline tests/integration/components/repo-refresh-status-test.js]: 🙌

File: tests/integration/components/projects/browser-family-selector-test.js (hunk 2 above)
Comment (cadeParade) [reply tests/integration/components/projects/browser-family-selector-test.js]: yes I did...now I have to go find it again 🤦‍♀️

**Pull request merged**